### PR TITLE
feat(mc): Add profile age, telemetryEnabled to snippets

### PR DESCRIPTION
**Pull request by k88hudson**
This PR adds information about profile age and telemetry to `gSnippetsMap` for use by snippet payloads.

### CODE REVIEW

File: system-addon/lib/SnippetsFeed.jsm [hunk 1]
```diff
@@ -5,43 +5,62 @@
...
 Cu.import("resource://gre/modules/Services.jsm");
-Cu.import("resource://gre/modules/Console.jsm");
 const {actionTypes: at, actionCreators: ac} = Cu.import("resource://activity-stream/common/Actions.jsm", {});
 
+XPCOMUtils.defineLazyModuleGetter(this, "ProfileAge",
+  "resource://gre/modules/ProfileAge.jsm");
+
 // Url to fetch snippets, in the urlFormatter service format.
 const SNIPPETS_URL_PREF = "browser.aboutHomeSnippets.updateUrl";
+const TELEMETRY_PREF = "datareporting.healthreport.uploadEnabled";
```
Comment (k88hudson) [inline system-addon/lib/SnippetsFeed.jsm]: Does this look like the right pref for the telemetry opt-out info you were looking for @glogiotatidis?

**Comment by as-pine-proxy** [comment]
[Treeherder: 9d347a6c01793fc352acf95cc58c68bd5ee952f3](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=9d347a6c01793fc352acf95cc58c68bd5ee952f3)

File: system-addon/lib/SnippetsFeed.jsm (hunk 1 above)
Comment (glogiotatidis) [inline system-addon/lib/SnippetsFeed.jsm]: Trying to find out. I also see that there's also  `datareporting.policy.dataSubmissionEnabled`. Sent an email to pdol to advise

Comment (glogiotatidis) [inline system-addon/lib/SnippetsFeed.jsm]: See also https://bugzilla.mozilla.org/show_bug.cgi?id=1384843

Comment (glogiotatidis) [inline system-addon/lib/SnippetsFeed.jsm]: https://github.com/mozilla/activity-stream/pull/3006

**Pull request merged**